src/styles/LessonSide.scss

```diff
@@ -1,7 +1,68 @@
 @use '_colors.scss' as colors;
 
+@mixin font-styles {
+  font-family: Quattrocento Sans, sans-serif;
+  font-size: 30px;
```

Comment:
Anywhere that uses pixels should be replaced with responsive units such as %, vw, or rem for font size.

Reply:
Not leaving comments for each one, but in general if you want your element to respond to different screen sizes, you shouldn't use pixels.

Reply:
I think `rem` is pretty standard for font size so I agree with that. 

Not sure about using `px` vs responsive for other parts though, thoughts @archishadatta ?

Reply:
Ok so I just cloned the repo and took a closer look. Here are the things that should be changed to responsive units; everything else can stay as pixels:

- Font Size: doesn't have to be a drastic change but should scale to different screen sizes. Use rem or calc(__ px + __vw)
- Height and Width of Button: for same reasons as font size
- Padding: more padding on bigger screen sizes. Also, you'll probably find later that 20px is too little padding for the sidebar.

Reply:
Also, the level-connector element seems to have a small gap in the middle where the background color shows through. The level numbers are also a bit off center in the circles. These are very minute issues so up to you whether you want to play around to fix it or not.